class RetryableException(Exception):
    pass


class ClientAuthError(Exception):
    pass


class ClientError(Exception):
    pass
